lms/static/sass/course/_profile.scss

```diff
@@ -179,6 +179,61 @@
       width: 100%;
     }
 
+    > .credit-eligibility{
+      border-top: 1px solid #e3e3e3;
+      margin-top: lh();
+      padding-left: 0;
+
+      > .credit-eligibility-container {
+          padding: lh();
+        > .credit-help {
+          background: #0078b0;
```

Comment:
Please use variables for any colours (defined here https://github.com/edx/edx-platform/blob/master/lms/static/sass/base/_variables.scss).